src/content/docs/faq.mdx

```diff
@@ -246,6 +246,18 @@ command works.
 [Uninstall](/uninstall) has details on how to remove Shorebird from your
 system should you choose.
 
+### Can the user delete patches from their device?
+
+Yes.
+
+Since patches are stored in the application data folder, if users
+go in the application settings inside their device system preferences and
+delete the app data, any stored patch will deleted as well.
+
+This should not break the application though. The application will be sent back to
+the original state of when the release was created and Shorebird will trigger
+an update to the latest patch next time the user open the app.
```

Comment:
I think this is Android-only.  I don't think there is any way to "clear application data" on iOS.

Reply:
Here is where we set the path on android (for your own amusement): https://github.com/shorebirdtech/engine/blob/a72fb61b0f4211313cabc1395e7ba062c5fa176a/shell/platform/android/flutter_main.cc#L130

Reply:
Fair enough! I checked on my iPhone and it seems that we really can't delete the data! TIL

I will mention that this is Android only